src/index.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
 
```

Comment:
this is one of the reason i say we should not use create react app cli tool, now that you removed service worker from the app, what happens if one day we all decide to use service worker or server side rendering? then we will have to inject and start a fresh configuration.. which you know can lead to problem